dracut.sh

```diff
@@ -2117,16 +2032,29 @@ fi
 
 if dracut_module_included "squash"; then
     dinfo "*** Squashing the files inside the initramfs ***"
+    declare squash_compress_arg
+    # shellcheck disable=SC2086
+    if [[ $compress ]]; then
+        if ! mksquashfs /dev/null "$DRACUT_TMPDIR"/.squash-test.img -no-progress -comp $compress &>/dev/null; then
```

Comment:
```suggestion
        if ! mksquashfs /dev/null "$DRACUT_TMPDIR"/.squash-test.img -no-progress -comp "$compress" &>/dev/null; then
```

Reply:
@ryncsn ping :point_up:

Reply:
Hi, actually I left it unquoted intentionally, from dracut man page:
```
--compress <compressor>
    ... If you pass it just the name of a compression program, it
           will call that program with known-working arguments. If you pass a
           quoted string with arguments, it will be called with exactly those
           arguments. 
```
So I also added `# shellcheck disable=SC2086` above.